lib/taskwarrior-web/services/builder/base.rb

```diff
@@ -6,13 +6,14 @@ module TaskwarriorWeb::CommandBuilder::Base
     :add => 'add',
     :update => TaskwarriorWeb::Config.version.major >= 2 ? ':id mod' : nil,
     :delete => 'rc.confirmation=no :id delete',
-    :query => TaskwarriorWeb::Config.version > '1.9.2' ? '_query' : 'export',
+    :query => 'export',
```

Comment:
Will this change break for older versions of taskwarrior? It looks like you've changed the logic considerably...